test/services/PrePopulationServiceSpec.scala

```diff
@@ -167,4 +188,2089 @@ object PrePopulationServiceSpec {
         )
       )
   }
+
+  val baseReturnJsValue: JsValue =
```

Comment:
Is there a better way to construct the base return / clean result value?
My concern is that if we change anything in the UA data structure (e.g. rename a field or move it to somewhere else in the structure) we would need to locate that in this json and manually correct it.

Reply:
Currently I'm copying this string (json) and pasting to an editor (in my IntelliJ json file) and fix/modify etc..
We can put this data in a json file and load it from there if we think this way will be easier. 

Producing this json data by building a complex scala object by setting case classes seems more complex to me.

What do you think @kberec ?